_src/pattern-library/sass/components/_forms.scss

```diff
@@ -0,0 +1,425 @@
+    display: inline-block;
+    width: auto;
+    vertical-align: bottom;
+}
+
+.wrapper-custom-radio,
+.wrapper-custom-checkbox {
+    display: inline-block;
+    z-index: z-index(base);
+
+    svg {
+        @include transition(all timing(xx-fast) ease-in-out 0s);
+        height: $baseline + px;
+        width: $baseline + px;
+        fill: $field-default-border;
+    }
+
+    &.is-selected {
+
+        svg {
+            fill: $field-default-border-focus;
+        }
+    }
+}
+
+.input-radio,
+.input-checkbox {
+
+    &:disabled,
+    &.is-disabled {
+
+        svg {
+            fill: $field-disabled-text-color;
+            cursor: default;
+        }
+    }
+
+    &.is-replaced {
+        position: absolute;
+        margin-top: ($baseline/4) + px;
+        z-index: z-index(mid-front);
+    }
+
+    &.is-transparent {
+        opacity: 0;
+    }
+}
+
+// ----------------------------
+// #TEXTAREA
+// ----------------------------
+.input-textarea {
+
+    &:focus,
+    &:active,
+    &.is-active {
+        border-color: $field-default-border-focus;
+        color: $field-default-text-color-hover;
+    }
+
+    &.textarea-short {
+        height: ($baseline*4.5) + px; // 3 clean lines
+    }
+
+    &.textarea-medium {
+        height: ($baseline*10) + px; // 8 clean lines
+    }
+
+    &.textarea-tall {
+        height: ($baseline*14.4) + px; // 12 clean lines
+    }
+
+    &.textarea-block {
+        width: 100%;
+    }
+}
+
+// ----------------------------
+// #SLIDERS
+// ----------------------------
+.input-slider,
+input[type="range"] {
+    -webkit-appearance: none;
+    height: ($baseline/3) + px;
+
+    // Hover states
+    &:hover,
+    &.is-hovered,
+    &:focus,
+    &.is-focused {
+        &::-webkit-slider-runnable-track {
+            background: $slider-default-hover;
+        }
+
+        &::-webkit-slider-thumb {
+            height: $baseline + px;
+            width: $baseline + px;
+            margin-top: -($baseline/3) + px;
+            background: $slider-default-handle-hover;
+        }
+
+        &::-moz-track {
+            background: $slider-default-hover;
+        }
+
+        &::-moz-range-thumb-thumb {
+            height: $baseline + px;
+            width: $baseline + px;
+            margin-top: -($baseline/3) + px;
+            background: $slider-default-handle-hover;
+        }
+
+        &::-ms-track {
+            background: $slider-default-hover;
+        }
+
+        &::-ms-thumb-thumb {
+            height: $baseline + px;
+            width: $baseline + px;
+            margin-top: -($baseline/3) + px;
+            background: $slider-default-handle-hover;
+        }
+    }
+
+    // Chrome track
+    &::-webkit-slider-runnable-track {
+        @include transition(all timing(xx-fast) ease-in-out 0s);
+        @include transform-origin(center center);
+        width: 100%;
+        height: ($baseline/2) + px;
+        border-radius: $slider-default-border-radius;
+        border: none;
```

Comment:
Is this needed for a reset? If so, perhaps it deserves 1) a comment that you're resetting it here, or 2) to be moved to a reset forms or general base reset part of our Sass?